pkg/sources/test_helpers.go

```diff
@@ -10,7 +10,7 @@ type ChunkFunc func(chunk *Chunk) error
 
 var MatchError = errors.New("chunk doesn't match")
 
-func HandleTestChannel(chunksCh chan *Chunk, cf ChunkFunc) error {
+func HandleTestChannel(chunksCh chan *Chunk, cf ChunkFunc, timeout int) error {
```

Comment:
**Nit (optional):** What you have here is good and I think it can be slightly improved by making `timeout` a `time.Duration`. That forces the user to include the unit at the callsite, which I think is clearer, and also allows for different units (milliseconds maybe?) if wanted.